uwrt_mars_rover_arm/uwrt_mars_rover_arm_hw/CMakeLists.txt

```diff
@@ -0,0 +1,107 @@
+cmake_minimum_required(VERSION 3.8)
+project(uwrt_mars_rover_arm_hw)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+
+add_custom_target(
+        copy-compile-commands ALL
+        ${CMAKE_COMMAND} -E copy_if_different
+        ${CMAKE_BINARY_DIR}/compile_commands.json
+        ${CMAKE_CURRENT_LIST_DIR}
+)
+
+
+
```

Comment:
What is this for?

Reply:
oh this was when I was trying to get my compile_commands.json to get produced ..... long story short I got it work and the cause of the issue was something else entirely .... I forgot to remove this

Reply:
the .clang-format file is already in the main branch of the repo. see how i setup this cmake function to use it in #171